src/instructlab/system/reset.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+# Standard
+import random
+import shutil
+
+# Third Party
+import click
+
+# First Party
+from instructlab import utils
+from instructlab.configuration import DEFAULTS
+
+
+def delete_storage_dirs():
+    """
+    Deletes all of the data in the instructlab storage & config directories.
+    """
+    click.echo(f"removing {DEFAULTS.CONFIG_HOME}...")
+    shutil.rmtree(DEFAULTS.CONFIG_HOME, ignore_errors=False)
+    click.echo(f"removing {DEFAULTS.DATA_HOME}...")
+    shutil.rmtree(DEFAULTS.DATA_HOME)
+    click.echo(f"removing {DEFAULTS.CACHE_HOME}...")
+    shutil.rmtree(DEFAULTS.CACHE_HOME)
+
+
+@click.command()
+@click.option(
+    "--skip-verify",
+    default=False,
+    help="Whether or not to skip verification of data deletion.",
+)
```

Comment:
Use this approach to implement a prompt: https://click.palletsprojects.com/en/8.1.x/options/#yes-parameters

Reply:
Filed for an issue for it [here](https://github.com/instructlab/instructlab/issues/1629), let's tackle it in a separate PR.